helm/values.yaml

```diff
@@ -0,0 +1,19 @@
+domain:
+  url: platform.debezium.io
+stage:
+  image: quay.io/debezium/platform-stage:latest
+api:
+  image: quay.io/debezium/platform-conductor:nightly
+kafka:
+  enabled: false
+database:
+  enabled: false
+  name: postgres
+  host: postgres
+  auth:
+    existingSecret: ""
+env:
+  QUARKUS_KUBERNETES_CLIENT_NAMESPACE: "{{ .Release.Namespace }}"
+  QUARKUS_HTTP_CORS_ORIGINS: "http://{{ .Values.domain.url }}"
+  CONDUCTOR_WATCHER_ENABLED: "true"
```

Comment:
this doesn't need to be configurable. Disabling this doesn't make sense. We should switch the default to true.

Reply:
Right.